src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './todo.scss';
+import PropTypes from 'prop-types';
+import { User } from '../User';
+
+export const Todo = ({ user, title, completed }) => (
+  <div className={completed ? 'box completed' : 'box noCompleted'}>
+    <User {...user} />
+    {title}
+    <p className={completed ? 'litleBox yes' : 'litleBox no'}>
+      {completed ? 'YES !!!' : 'NO !!!'}
```

Comment:
```suggestion
      {completed ? 'yes' : 'no'}
```
use text-transform css property to uppercase text